src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -316,15 +307,13 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
           wip: obj.clientRemittance?.wip,
           workAddressId: obj.clientRemittance?.workAddressId,
           title: obj.clientRemittance?.title,
-          mobileVerified: obj.clientRemittance?.mobileVerifiedStatus,
+          mobileVerified: obj.clientRemittance?.mobileVerificationStatus,
```

Comment:
remove this since you removed the mobile verification field from the form.

Reply:
but i need when to send value  0 not null